lib/aqua/Taskfile.yaml

```diff
@@ -0,0 +1,17 @@
+version: "3"
+
+tasks:
+  install:
+    desc: Installs the Aqua CLI using the universal installer script, sets in PATH, and upgrades.
+    silent: true
+    cmds:
+      - |
```

Comment:
The Styleguide recommends to [prefer external scripts over complex multi-line commands](https://taskfile.dev/styleguide/#prefer-external-scripts-over-complex-multi-line-commands). This one is relatively simple, but I think we should start following this pattern in the lib for consistency.

Reply:
Yeah and I agree with that... to a point. I feel like a script that is less than 30 lines... should be just a block command. If it's over 30 lines then it's probably good to break it out into a script. How does that sound for our own internal style?

Reply:
Sounds good 👍